var/spack/repos/builtin/packages/mercury/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class Mercury(CMakePackage):
+    """Mercury is a C library for implementing RPC, optimized for HPC"""
+
+    homepage = 'https://mercury-hpc.github.io/'
+    url = 'https://github.com/mercury-hpc/mercury/archive/0.9.0.tar.gz'
+    git = 'https://github.com/mercury-hpc/mercury.git'
+
+    version('develop', branch='master', submodules=True)
+    version('1.0.0', tag='v1.0.0', submodules=True, preferred=True)
```

Comment:
No need to mark this as preferred, Spack already defaults to the most recent numerical release.